specification/storage/data-plane/Microsoft.BlobStorage/preview/2020-10-02/blob.json

```diff
@@ -7713,6 +7719,9 @@
               "$ref": "#/definitions/StorageError"
             }
           }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "NextMarker"
```

Comment:
How do customers opt-in to this behavior?  Is it only returned when marker or maxresults is provided?

This might end up causing problems with code gen - in which case maybe we need to split this apart into separate operations (one legacy "here's everything" and a new pageable alternative).

Reply:
That is correct.  If customers don't provide `maxresults` or `marker`, the service defaults to the old behavior.

I actually haven't had an issue in the .NET code gen, since these parameters are optional.